contrib/ros/robots/sawyer.py

```diff
@@ -2,14 +2,13 @@
 Sawyer Interface
 """
 
-import gym
+from gym.spaces import Box
```

Comment:
There is also a `garage.Box`, so it is best to always use the `gym` prefix (e.g. refer to it as `gym.spaces.Box`)